src/lightning_fabric/fabric.py

```diff
@@ -148,6 +157,16 @@ def is_global_zero(self) -> bool:
         """Whether this rank is rank zero."""
         return self._strategy.is_global_zero
 
+    @property
+    def loggers(self) -> List[Logger]:
+        """Returns all loggers passed to Fabric."""
+        return self._loggers
+
+    @property
+    def logger(self) -> Logger:
```

Comment:
shall this have index as an argument so not to be a property?

Reply:
This can already be accessed like so through an index:

```py
fabric.loggers[2]
```

and 

```py
fabric.logger == fabric.loggers[0]
```

Or did you mean something else?